### PR TITLE
Docs: Contribution guidelines for updating Drone pipelines

### PR DESCRIPTION
**Pull request by tskarhed**
**What is this feature?**

Documentation for 

**Why do we need this feature?**

Easier to get started with pipeline development, and reduces questions to grafana-delivery

**Who is this feature for?**

Grafana contributors who wants to create new pipeline steps

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: contribute/drone-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Making changes to the Drone pipeline
+
+_Only people in the Grafana organization can make changes to the Drone pipeline_
+
+The Drone pipelines are built with [Starlark](https://github.com/bazelbuild/starlark), a language which is similar to Python. The Starlark files are located in [`scripts/drone`](https://github.com/grafana/grafana/tree/main/scripts/drone).
+
+## Setup
+- Set environment variables `DRONE_SERVER` and `DRONE_TOKEN`, which can be found on your [Drone account](https://drone.grafana.net/account). These are used to verify that onyl Grafana employees can make changes to the pipelines.
```
Comment (zerok) [inline contribute/drone-pipeline.md:8]:
```suggestion
- Set environment variables `DRONE_SERVER` and `DRONE_TOKEN`, which can be found on your [Drone account](https://drone.grafana.net/account). These are used to verify that only Grafana employees can make changes to the pipelines.
```

File: contribute/drone-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Install [buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md), and use it to format the Starlark files you edit.
+
+## Develop
+- Open a PR where you can do test runs for your changes. If you need to experiment with secrets, create a PR in the [grafana-ci-sandbox repo](https://github.com/grafana/grafana-ci-sandbox), before opening a PR in the main repo.
+- Run `make drone` after making changes to the Starlark files. This builds the `.done.yml` file.
```
Comment (zerok) [inline contribute/drone-pipeline.md:13]:
```suggestion
- Run `make drone` after making changes to the Starlark files. This builds the `.drone.yml` file.
```

**Review by zerok — APPROVED** [review]
Thank you :)

**Review by zerok — CHANGES_REQUESTED** [review]
Just found some minor typos :)

File: contribute/drone-pipeline.md (hunk 1 above)
Comment (tskarhed) [reply contribute/drone-pipeline.md:8]: I should read what I write :facepalm:

**Review by zerok — APPROVED** [review]

**Comment by tskarhed** [comment]
@grafana/docs-grafana Any input here?

**Pull request merged**